gcloud/exceptions.py

```diff
@@ -38,7 +39,7 @@ def __init__(self, message, errors=()):
         super(GCloudError, self).__init__()
         # suppress deprecation warning under 2.6.x
         self.message = message
-        self._errors = [error.copy() for error in errors]
+        self._errors = errors
```

Comment:
In the `errors` property a copy is done again. It was redundant. I noticed that copy because I replaced it with `copy.deepcopy`.